code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -541,6 +542,37 @@ nobiliumsuppression = INFINITY
 			air.temperature = max(((temperature * old_heat_capacity + energy_used) / new_heat_capacity), TCMB) //the air heats up when reacting
 		return REACTING
 
+/datum/gas_reaction/nitryl_decomposition //The decomposition of nitryl. Exothermic. Requires oxygen as catalyst.
+	priority = 2
+	name = "Nitryl Decomposition"
+	id = "nitryl_decomp"
+
+/datum/gas_reaction/nitryl_decomposition/init_reqs()
+	min_requirements = list(
+		/datum/gas/oxygen = MINIMUM_MOLE_COUNT,
+		/datum/gas/nitryl = MINIMUM_MOLE_COUNT,
+		"MAX_TEMP" = 600
```

Comment:
`"MAX_TEMP"` stops the reaction if the temperature is over 600 K, you'd want just `"TEMP"`